bigquery/google/cloud/bigquery/client.py

```diff
@@ -1929,6 +1935,61 @@ def list_rows(
         )
         return row_iterator
 
+    def _schema_from_json_file_object(self, file):
+        """Helper function for schema_from_json that takes a
+           file object that describes a table schema.
+
+           Returns:
+                List of schema field objects.
+        """
+        schema_field_list = list()
+        json_data = json.load(file)
+
+        for field in json_data:
+            schema_field = SchemaField.from_api_repr(field)
+            schema_field_list.append(schema_field)
+
+        return schema_field_list
+
+    def schema_from_json(self, file_or_path):
+        """Takes a file object or file path that contains json that describes
+            a table schema.
```

Comment:
Nit: This and the other docstrings (except for the first line) look a like they are indented 4 spaces too many.

Reply:
Fixed indentations.